src/redux/books/books.js

```diff
@@ -0,0 +1,15 @@
+export default function booksReducer(state = {}, action) {
+  switch (action.type) {
+    case 'ADD_BOOK': return {};
+    case 'REMOVE_BOOK': return {};
+    default: return state;
+  }
+}
+
+export function addBook() {
+  return { type: 'ADD_BOOK' };
+}
+
+export function removeBook() {
+  return { type: 'REMOVE_BOOK' };
+}
```

Comment:
- [ ] It would be better if the value of each action type follows the `ducks` pattern. For example; instead of `ADD_BOOK` `bookstore/books/ADD_BOOK`, let us kindly do same for our `REMOVE_BOOK` action type